### ISSUE TITLE
Click on a bar graph - get a listing of movies/shows with that stat

### ISSUE THREAD

**Issue by diedrichg** [mregni/EmbyStat#824]
# Feature request

**Describe the solution you'd like
I'd love to see the graphs and a few of the blue headers be clickable items. For example:
If I'm on the Movies page and I click on the "Comedy" bar in the 'Genre' graph, I would like to be taken to a page with a listing of all those movies that are in the Comedy genre.

**It would be really cool**, instead, if the graph area would be expanded below the graph so that you would see the list of movies directly below that graph - this way you would not leave the Movies page and you could then simply click on "Comedy" again to 'close' the list or click on another genre to bring up that list of movies. This request goes for all graphs (genre, rating, official rating, year, etc.).

**Comment by mregni** [comment]
Sounds cool indeed! 

Displaying the list of movies directly under the graph will have a lot of design issues I think. What you think about a clickable bar that will direct you to a new page. But on top of the new page, the graph is displayed. That way you can still navigate between the bars in the graph (switching from "Action" to "Comedy" movies). I just have to figure out an elegant way to go back to the movies page (or show)

**Comment by diedrichg** [comment]
Agreed, taking the user to a new page with that graph as the sole graph on the page would be the simplest implementation. This way the user would be unaware they are leaving the graph page each time they click a bar. Yes, finding a way to navigate back to the movies page would be essential. For now, you have the main menu on the left menu bar. For ease of access and keeping the mouse cursor from moving too far, you could create a new menu header (tabs?) that has the names of the graphs that you could navigate to quickly. This would get messy though once you have a long list of graphs available.

**Comment by mregni** [comment]
Maybe I can create a sub-menu called "Graph details" are something like that and add a dropdown in the title header of the page so a user can switch between graphs as well without leaving the page. And then you won't get a massive list of sub-menus for each graph in the menu. Just an idea
